src/Model/LoginSession.php

```diff
@@ -156,7 +164,7 @@ public function canDelete($member = null)
      * @param Member $member
      * @return bool
      */
-    public function handlePermission(string $funcName, $member): bool
+    private function handlePermission(string $funcName, $member): bool
```

Comment:
Don't think there's a uses case where this needs to be called from outside the class so I made it private